test-series/1/test-1.0.0.ipynb

```diff
@@ -0,0 +1,1449 @@
+{
```

Comment:
<p>It would be helpful to see the fg line at high k_par. It's unclear if this shows sum = fg + eor.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>I'll clarify in-text that sum is fg + eor, but I'm not sure that expanding the plot y-range is a good idea.</p>